### DISCUSSION TITLE
Ability to select the whole group on MultiSelect component

### DISCUSSION THREAD

**Discussion by aprilmintacpineda** [mantinedev/mantine#2315] [Feature requests]
on the docs https://mantine.dev/core/multi-select/#group-items

![image](https://user-images.githubusercontent.com/21032419/187927055-bdfee933-5e70-41e2-8505-8b3822635e30.png)

I should be able to click on the group label to select a whole group.

**Comment by zhulien-ivanov** [comment]
You're looking for https://github.com/mantinedev/mantine/discussions/1051. It's in the backlog, as far as I know.

**Reply by aprilmintacpineda** [reply]
No, that's a different component.

**Reply by aprilmintacpineda** [reply]
Actually looking at the demo it does seem to fulfill that, only it has quite a different behavior from MultiSelect.

**Reply by zhulien-ivanov** [reply]
It has exactly the same behaviour but provides more functionality. It allows you to select multiple options from a set and also the option for categorization. Basically, `MultiSelect` is a handicapped variation of `TreeSelect` but with "hidden checkboxes" and not allowing deeper nesting of items.
